engine/runtime-compiler/src/main/scala/org/enso/compiler/Compiler.scala

```diff
@@ -873,12 +865,12 @@ class Compiler(
     * @param source the original source code.
     * @param inlineContext the inline compilation context.
     */
-  def runErrorHandlingInline(
+  private def runErrorHandlingInline(
     ir: Expression,
     source: Source,
     inlineContext: InlineContext
   ): Unit =
-    if (config.isStrictErrors) {
+    if (inlineContext.compilerConfig.isStrictErrors) {
```

Comment:
Looks like a bug, right?

Reply:
Yes, otherwise it ignores the compiler config of the inline context argument.